### DISCUSSION TITLE
Email notification on vulnerability update

### DISCUSSION THREAD

**Discussion by nickthebeer** [DependencyTrack/dependency-track#3130] [Q&A] [answered]
Hi,

I can't see anyone else asking this question and I'm struggling to confirm this with DT but what happens with regards to email notifications when a vulnerability status changes. Eg: I get an email when a project is affected by a new vulnerability but what happens when a field is updated (for example the severity) - does this generate another email?

**Comment by nscuro** [comment, marked as answer]
There currently is no such notification. A variation of it has been discussed here: https://github.com/DependencyTrack/dependency-track/issues/2361

**Reply by nickthebeer** [reply]
Ah perfect, thank you, I missed that issue when searching.
